core/src/main/java/discord4j/core/spec/AuditSpec.java

```diff
@@ -18,10 +18,22 @@
 
 import reactor.util.annotation.Nullable;
 
+/** A spec which can optionally have a reason in the audit logs when built. */
 public interface AuditSpec<T> extends Spec<T> {
 
+    /**
+     * Sets the audit log reason to show in the audit logs when the spec is built.
```

Comment:
I would word it "Sets the reason to show in the audit logs when the spec is built."